src/plugins/tables/tables.js

```diff
@@ -84,6 +84,29 @@ var componentName = "wb-tables",
 						dataTableExt = $.fn.dataTableExt;
 
 					/*
+					* Add printing support
+					*/
+					if ( $elm.hasClass( "wb-tables-print" ) ) {
+						var $btn = $( "<button class='btn btn-default mrgn-bttm-md' id='" + elmId + "_datatable-print'>Print Table</button>" );
```

Comment:
if it is not useful/needed for this fix, so remove it because it will save processing time and reduce the wet-boew js file size.